doc/source/whats_new/v0.9.0.txt

```diff
@@ -63,6 +63,8 @@ The following changes may break old user code.
      event['descriptor'] == descriptor['uid']
      descriptor['run_start'] == run_start['uid']
      run_stop['run_start'] == run_start['uid']
+* The type of db.filters changed from list to dict.
```

Comment:
This is going to generate sphinx errors. You need a blank line between the end of the ``.. code-block`` above and this bullet point.